src/parser/mod.rs

```diff
@@ -7646,6 +7646,39 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_index_exprs(&mut self) -> Result<Vec<IndexExpr>, ParserError> {
+        self.parse_parenthesized(|p| p.parse_comma_separated(Parser::parse_index_expr))
+    }
+
+    pub fn parse_index_expr(&mut self) -> Result<IndexExpr, ParserError> {
```

Comment:
Since this is flagged as a public function can we add a short description mentioning what it does?